package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Order_Tracking",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
It's good practice to specify the author of the application. This can be either an individual or an organization.